internal/oidc/keyhandler_test.go

```diff
@@ -77,7 +83,7 @@ func TestNewKeyHandler(t *testing.T) {
 	keyID3, err := getKeyIDFromTokenString(token3.AccessToken)
 	require.NoError(t, err)
 	op.Close(t)
-	_, err = keyHandler.getKeyFromID(ctx, keyID3)
+	_, err = keyHandler.getKeyFromID(ctx, keyID3, "")
 	require.Error(t, err)
 }
```

Comment:
I would like to add a new test that has two keys with the same `kid` but different `alg` and make sure this works and continues to work in the future.